bin/yml2json

```diff
@@ -5,4 +5,8 @@ require 'erb'
 require 'json'
 require 'yaml'
 
+def git_desc
+  ENV.fetch('GIT_DESC', `git describe --always --dirty --tags`.strip)
```

Comment:
I presume the reason for duplicating this here and in `.travis.yml` is that this instance is required locally, but the `.travis.yml` instance avoids calling out to git N times in each Travis job?